Jenkinsfile_CNP

```diff
@@ -10,8 +10,27 @@ def product = "ccd"
 def component = "case-print-service"
 
 def secrets = [
+    'ccd-${env}': [
+        secret('ccd-caseworker-autotest-email', 'CCD_CASEWORKER_AUTOTEST_EMAIL'),
```

Comment:
could we please not use email addresses from secrets? We can hard code the emails into the value of an env variable.